rules/Php81/Rector/FuncCall/NullToStrictStringFuncCallArgRector.php

```diff
@@ -477,10 +474,6 @@ private function processNullToStrictStringOnNodePosition(
             return null;
         }
 
-        if ($this->isAnErrorTypeFromParentScope($argValue, $scope)) {
```

Comment:
this should be reverted, most ArrayDimFetch without context will go ErrorType, and it wil cause huge string cast in legacy code